### ISSUE TITLE
Attempting to install PECL uuid extension fails

### ISSUE THREAD

**Issue by ramsey** [phpbrew/PEARX#5]
I'm using phpbrew's extension installation command to attempt to install pecl/uuid, and it's failing:

```
$ phpbrew ext install uuid
===> Downloading from http://pecl.php.net/get/uuid-1.0.3.tgz
[==================================================================] 6.18/6.18KB 100%
===> /home/vagrant/.phpbrew/distfiles/uuid-1.0.3.tgz downloaded.
===> Extracting to /home/vagrant/.phpbrew/build/php-5.6.6/ext...
invalid pear dependency type.
```

I assume the "invalid pear dependency type." message is coming from this project.

If I try installing with the `pecl` command (`pecl install uuid`), it installs just fine.

**Comment by borgihalim** [comment]
Dear Ramsey,

I tried to install it with this command pecl install uuid but i received this error : 
downloading uuid-1.0.4.tgz ...
Starting to download uuid-1.0.4.tgz (14,900 bytes)
.....done: 14,900 bytes
could not extract the package.xml file from "/build/php5-0LI9sl/php5-5.5.9+dfsg/pear-build-download/uuid-1.0.4.tgz"
Download of "pecl/uuid" succeeded, but it is not a valid package archive
Error: cannot download "pecl/uuid"
Download failed
install failed

I'm using Ubuntu 14.04
Thank you
Halim EL BORGI

**Comment by morozov** [comment]
Closing as fixed by #9.
